office365_admin/help.md

```diff
@@ -171,16 +210,29 @@ This action is used to remove a user from Office365.
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|success|boolean|False|return true if it worked|
+|success|boolean|False|Return true if it worked|
 
 Example output:
-
 ```
 {
   "success": true
 }
 ```
 
+#### Get Subscribed SKUs
```

Comment:
I did not add this action in the PR.  I used icon-plugin generate help ...

Reply:
The order of the items was changed.

Reply:
If it's missing added. If you can, update the example output please.